client/ayon_maya/plugins/load/load_layout.py

```diff
@@ -0,0 +1,231 @@
+from maya import cmds
+import math
+import json
+import collections
+import ayon_api
+from ayon_maya.api import plugin
+from ayon_maya.api.lib import unique_namespace
+from ayon_core.pipeline import (
+    load_container,
+    get_representation_path,
+    discover_loader_plugins,
+    loaders_from_representation,
+    get_current_project_name
+)
+from ayon_maya.api.pipeline import containerise
+
+
+class LayoutLoader(plugin.Loader):
+    """Layout Loader(json)"""
+
+    product_types = {"layout"}
+    representations = {"json"}
+
+    label = "Layout Loader(json)"
+    order = -10
+    icon = "code-fork"
+    color = "orange"
+
+    def _get_repre_entities_by_version_id(self, data):
+        version_ids = {
+            element.get("version")
+            for element in data
+        }
+        version_ids.discard(None)
+
+        output = collections.defaultdict(list)
+        if not version_ids:
+            return output
+
+        project_name = get_current_project_name()
+        repre_entities = ayon_api.get_representations(
+            project_name,
+            representation_names={"fbx", "abc"},
+            version_ids=version_ids,
+            fields={"id", "versionId", "name"}
+        )
+        for repre_entity in repre_entities:
+            version_id = repre_entity["versionId"]
+            output[version_id].append(repre_entity)
+        return output
+
+    @staticmethod
+    def _get_loader(loaders, product_type):
+        name = ""
+        if product_type in {
+            "rig", "model", "camera",
+            "animation", "staticMesh",
+            "skeletalMesh"}:
+                name = "ReferenceLoader"
+
+        if name == "":
+            return None
+
+        for loader in loaders:
+            if loader.__name__ == name:
+                return loader
+
+        return None
+
+    def get_asset(self, instance_name):
+        container = [
+            con for con in cmds.ls(f"{instance_name}*")
+            if con.endswith("_CON")
+        ][0]
+        namespace = cmds.getAttr(f"{container}.namespace")
+        asset = [asset for asset in cmds.ls(f"{namespace}:*", assemblies=True)][0]
+        return asset
+
+    def _process(self, filepath, options, loaded_options=None):
+
+        with open(filepath, "r") as fp:
+            data = json.load(fp)
+
+        all_loaders = discover_loader_plugins()
+
+        if not loaded_options:
+            loaded_options = []
+
+        repre_entities_by_version_id = self._get_repre_entities_by_version_id(
+            data
+        )
+        for element in data:
```

Comment:
Can we get some hints or comments on what we expect `data` to be so that just by reading the code we can have some expectancy of what we assume is in the json?